src/loadexec/exports.exp

```diff
@@ -54,9 +54,10 @@ PSP_EXPORT_END
 PSP_EXPORT_START(LoadExecForUser, 0x0011, 0x4001)
 PSP_EXPORT_FUNC_HASH(sceKernelExitGame)
 PSP_EXPORT_FUNC_HASH(sceKernelExitGameWithStatus)
+# sceKernelSafetyCheck1 according to ps3 NIDs, but doesn't match SHA1
 PSP_EXPORT_FUNC_NID(LoadExecForUser_362A956B, 0x362A956B)
 PSP_EXPORT_FUNC_HASH(sceKernelRegisterExitCallback)
-PSP_EXPORT_FUNC_NID(LoadExecForUser_8ADA38D3, 0x8ADA38D3)
+PSP_EXPORT_FUNC_HASH(sceKernelLoadExecNpDrm)
```

Comment:
As I am still using mainly PRXTool for RE'ing and we don't really have a comprehensive NID table for 6.60, I typically check the exports.exp file of a module if I need to to map one of its NIDs to a function name. For this, keeping the NID ext to the name really comes in handy a lot. I've checked PRXTool and it appears to me I can only specify a NID table as an xml file and cannot make use of an exports.exp file to map NIDs to names. As such, I would like to see that we use keep using `PSP_EXPORT_FUNC_NID` even if the NID has been cracked:
```
PSP_EXPORT_FUNC_NID(sceKernelLoadExecNpDrm, 0x8ADA38D3)
```

That's how I have been doing it for power too. See https://github.com/uofw/uofw/blob/power/src/power/exports.exp#L12

Reply:
I don't have a strong opinion against putting the NIDs in all the exports.exp files (although I think it's a good idea to keep somewhere the information that it indeed matches the name), however I don't think this issue is where we should do it. I just made the NID names I added homogeneous with the rest.

Reply:
I don't think it really matters whether we are consistent in an exports.exp file or not. So while I wouldn't ask you to change every FUNC_HASH() to a FUNC_NID() as part of this PR, I would prefer to at least use FUNC_NID() for the updated NIDs. Who knows when in the future someone will have the time and mood to update all FUNC_HASH() to FUNC_NID(). In the meantime, I basically update FUNC_HASH() to FUNC_NID() as I encounter them in my [reverse engineering](https://github.com/uofw/uofw/pull/82/files?file-filters%5B%5D=.exp#diff-dcd4201b3042e5e9c5fd396c26ab5db8155bac170971d772c701ee1c5b8ea3a8R31).

Out of scope of this PR, this makes the point for an update NID database since the existing ones are heavily outdated. If we had a maintained NID database, then I wouldn't have to lookup NIDs in our export-exp files. Perhaps this should get its own issue in the repo for tracking...